packages/web-app-files/src/helpers/resources.ts

```diff
@@ -49,17 +49,20 @@ export function buildResource(resource): Resource {
   }
 
   const id = resource.fileInfo[DavProperty.FileId]
+  const name = resource.fileInfo[DavProperty.Name]
```

Comment:
The three lines can be shortened to
```suggestion
  const name = resource.fileInfo[DavProperty.Name] || path.basename(resource.name)
```